spec/hutch/worker_spec.rb

```diff
@@ -45,9 +45,17 @@
     end
 
     it 'sets up a subscription' do
-      expect(queue).to receive(:subscribe).with(manual_ack: true)
+      expect(queue).to receive(:subscribe).with(consumer_tag: %r(^hutch\-.{36}$), manual_ack: true)
```

Comment:
That would be a cool touch.

(Then we could figure out ways to validate configuration options. Example addition to the Config options' DSL: `max_length: 34`.)